lib/libvol2bird.h

```diff
@@ -11,7 +11,9 @@
  *
  */
 
+#ifndef NOCONFUSE
```

Comment:
My current makefiles don't implement this flag, where should I add it?

Reply:
For standalone variant we asume that confuse is used so that flag should not be defined so this should not be defined for vol2bird.